src/interfaces/IConstantProductPair.sol

```diff
@@ -1,10 +1,8 @@
 pragma solidity 0.8.13;
 
-import { IAssetManager } from "src/interfaces/IAssetManager.sol";
-import { IAssetManagedPair } from "src/interfaces/IAssetManagedPair.sol";
 import { GenericFactory } from "src/GenericFactory.sol";
 
-interface IConstantProductPair is IAssetManagedPair {
+interface IConstantProductPair {
```

Comment:
Why did we remove the inheritance here? Does doing it here mean the discrete pairs are not compatible?

Reply:
since we're changing the interface in the very near future, this has become irrelevant, resolving for now.